shesmu-pluginapi/src/main/java/ca/on/oicr/gsi/shesmu/plugin/filter/BaseAgoActionFilter.java

```diff
@@ -42,4 +42,13 @@ public final long getOffset() {
   public final void setOffset(long offset) {
     this.offset = offset;
   }
+
+  @Override
+  public String toString() {
+    StringBuilder writeOut = new StringBuilder();
+    writeOut.append(getOperation()).append(" filter of offset: ").append(offset);
+    return writeOut.toString();
+  }
+
+  protected abstract String getOperation();
```

Comment:
Would logging the class name here be just as useful too since these are all defined basically at the class level?

Reply:
Fixed